src/index.ts

```diff
@@ -48,21 +49,25 @@ function extractBase64Data(pathname: string | null): string | null {
 }
 
 export function fromFeedSigil(sigil: FeedId) {
+  if (!ref.isFeed(sigil)) return null;
```

Comment:
Could replace this with just a simple check

```suggestion
  if (typeof sigil !== 'string') return null;
  if (!sigil.startsWith('@')) return null;
```